misk-hibernate/src/main/kotlin/misk/jdbc/DataSourceConfig.kt

```diff
@@ -194,10 +194,14 @@ data class DataSourceConfig(
           useSSL = true
         }
 
-        if (useSSL) {
-          queryParams += "&useSSL=true"
-          queryParams += "&requireSSL=true"
+        val sslMode = if (useSSL && verify_server_identity) {
+          "VERIFY_IDENTITY"
+        } else if (useSSL) {
+          "VERIFY_CA"
+        } else {
+          "PREFERRED"
```

Comment:
Don't we require all our servers to have SSL enabled? PREFERRED will default to unencrypted connection if server hasn't enabled it

Reply:
We don’t enable SSL in local dev, so it’s a matter of which set of config files we add overrides to. Setting to PREFERRED as default preserves current behavior.